piq/gmsd.py

```diff
@@ -29,25 +29,59 @@ def _gmsd(prediction: torch.Tensor, target: torch.Tensor,
             ``'none'`` | ``'mean'`` | ``'sum'``. ``'none'``: no reduction will be applied,
             ``'mean'``: the sum of the output will be divided by the number of
             elements in the output, ``'sum'``: the output will be summed.
+        data_range: The difference between the maximum and minimum of the pixel value,
+            i.e., if for image x it holds min(x) = 0 and max(x) = 1, then data_range = 1.
+            The pixel value interval of both input and output should remain the same.
+
+    Returns:
+        gmsd : Gradient Magnitude Similarity Deviation between given tensors.
+
+    References:
+        https://arxiv.org/pdf/1308.3052.pdf
+    """
+
+    _validate_input(input_tensors=(prediction, target), allow_5d=False, scale_weights=None)
+    prediction, target = _adjust_dimensions(input_tensors=(prediction, target))
+
+    if data_range == 255:
+        prediction = prediction / 255.
+        target = target / 255.
```

Comment:
Let's to range conversion the same way it's done in VSI and other more recent metrics.
```suggestion
    prediction = prediction / float(data_range)
    target = target / float(data_range)
```